### PR TITLE
various eslint fixes

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: mocha-multi.js [hunk 1]
```diff
@@ -56,12 +60,13 @@ MochaMulti.prototype.done = function (failures, fn) {
...
     debug('Awaiting on %j reporters to invoke done callback.', count);
     const cb = () => {
-      if (--count === 0) {
+      if (count <= 0) {
         debug('All reporters invoked done callback.');
         fn(failures);
       } else {
         debug('Awaiting on %j reporters to invoke done callback.', count);
       }
+      count -= 1;
```
Comment (glenjamin) [inline mocha-multi.js]: this is a logic change, the count used to happen before the condition.

File: mocha-multi.js [hunk 2]
```diff
@@ -74,17 +79,16 @@ MochaMulti.prototype.done = function (failures, fn) {
 };
 
 const msgs = {
-  no_definitions: "reporter definitions should be set in \
-the `multi` shell variable\n\
-eg. `multi='dot=- xunit=file.xml' mocha`",
-  invalid_definition: "'%s' is an invalid definition\n\
-expected <reporter>=<destination>",
+  no_definitions: 'reporter definitions should be set in ' +
```
Comment (glenjamin) [inline mocha-multi.js]: should probably use a template string here?

File: mocha-multi.js (hunk 1 above)
Comment (graingert) [reply mocha-multi.js]: ah of course it did.

File: mocha-multi.js (hunk 2 above)
Comment (graingert) [reply mocha-multi.js]: no, template strings preserve source newlines.

Comment (glenjamin) [reply mocha-multi.js]: aha, of course.

**Comment by graingert** [comment]
@glenjamin I'm happy with this batch.

**Comment by glenjamin** [comment]
Great stuff, thanks.

Not sure I'll have time for more reviews today, but might check back later.

**Comment by graingert** [comment]
@glenjamin oh, sorry I was saving the worst till last!

**Pull request merged**